app/models/effective/datatables/logs.rb

```diff
@@ -37,11 +55,17 @@ class Logs < Effective::Datatable
         # A nil attributes[:log_id] means give me all the top level log entries
         # If we set a log_id then it's for sub logs
         def collection
-          if attributes[:user_id].present?
-            Effective::Log.unscoped.where(:parent_id => attributes[:log_id]).where(:user_id => attributes[:user_id]).includes(:user)
-          else
-            Effective::Log.unscoped.where(:parent_id => attributes[:log_id]).includes(:user)
-          end
+          col = Effective::Log.unscoped
+            .where(parent_id: attributes[:log_id])
+            .joins('LEFT JOIN users ON users.id = logs.user_id')
```

Comment:
@andie-chernysh, thanks for all the good info about the internals of `::includes` vs `::joins`.  I didn't know that includes would perform 2 operations.  I did some quick research and found a blog post about this topic that makes it sound like Rails decides whether to use a join or use two smaller queries for memory consideration.  The post concludes that `::includes` is probably better to use most of the time.

Finally, I did originally think it would perform 2 queries because of the `#where` on line 63.